__tests__/components/TransactionList/TransactionRow.spec.ts

```diff
@@ -0,0 +1,888 @@
+            UInt64.fromUint(10),
+            signedTransaction,
+            NetworkType.TEST_NET,
+            UInt64.fromUint(100),
+            mockSignature,
+            PublicAccount.createFromPublicKey(currentSigner.publicKey, NetworkType.TEST_NET),
+        );
+
+        const preparedData = [
+            {
+                transactionType: 'other transaction deadline in 6 hours',
+                mockTransaction: mockTransferTransaction(),
+                expectedDate: '2019-11-10 22:00:00',
+            },
+            {
+                transactionType: 'Aggregate Bond transaction deadline in 48 hours',
+                mockTransaction: mockAggregateTransaction(),
+                expectedDate: '2019-11-09 00:00:00',
+            },
+            {
+                transactionType: 'Hash Lock transaction deadline in 6 hours',
+                mockTransaction: hashLockTransaction,
+                expectedDate: '2019-11-10 18:00:00',
+            },
+        ];
+
+        preparedData.map((test) => runTransactionDateTests(test));
+    });
+
+    describe('isOptinPayoutTransaction', () => {
+        test('returns false when transaction is not exist', async () => {
+            // Arrange:
+            const wrapper = getTransactionRowWrapper(
+                { currentProfile: testnetProfile, currentAccount: WalletsModel1 },
+                { transaction: mockTransferTransaction() },
+            );
+
+            const vm = wrapper.vm as TransactionRowTs;
+
+            vm.transaction = undefined;
+
+            // Act + Assert:
+            expect(vm.isOptinPayoutTransaction).toBe(false);
+        });
+
+        const runIsOptinPayoutTransactionTests = (state) => {
+            // Arrange:
+            const wrapper = getTransactionRowWrapper(state, { transaction: mockAggregateTransaction() });
+
+            const vm = wrapper.vm as TransactionRowTs;
+
+            // Act + Assert:
+            expect(vm.isOptinPayoutTransaction).toBe(true);
+        };
+
+        test('returns true when transaction announced by NGL (testnet)', async () => {
+            runIsOptinPayoutTransactionTests({ currentProfile: testnetProfile, currentAccount: WalletsModel1 });
+        });
+
+        test('returns true when transaction announced by NGL (mainnet)', async () => {
+            runIsOptinPayoutTransactionTests({ currentProfile: getTestProfile('profile_mainnet'), currentAccount: WalletsModel2 });
+        });
+    });
+
+    test('returns explorer url link', () => {
+        // Arrange:
+        const wrapper = getTransactionRowWrapper({ currentProfile: testnetProfile }, { transaction: mockTransferTransaction() });
+        const vm = wrapper.vm as TransactionRowTs;
+
+        // Act + Assert:
+        expect(vm.explorerUrl).toBe(
+            'https://testnet.symbol.fyi/transactions/3A4B36EDFD3126D3911916497A9243336AE56B60B5CEB9410B4191D7338201CD',
+        );
+    });
+
+    describe('hasMissSignatures', () => {
```

Comment:
i assume Miss is short for Missing ? please use full word everywhere?

Reply:
agree, I will patch that change.